packages/graphback-core/src/plugin/GraphbackCoreMetadata.ts

```diff
@@ -28,7 +28,7 @@ export class GraphbackCoreMetadata {
 
     public constructor(globalConfig: GraphbackGlobalConfig, schema: GraphQLSchema) {
         this.schema = schema;
-        this.supportedCrudMethods = Object.assign(defaultCRUDGeneratorConfig, globalConfig.crudMethods)
+        this.supportedCrudMethods = Object.assign({}, defaultCRUDGeneratorConfig, globalConfig.crudMethods)
```

Comment:
This can be reverted

Reply:
This will work only if a single metadata object is present. 
Having multiple metadata objects will still fail. 
Will add test for that case.

Reply:
Unit tests now cover this case as well